roles/ceph-handler/handlers/main.yml

```diff
@@ -48,10 +48,10 @@
       listen: "restart ceph mons"
       when:
         # We do not want to run these checks on initial deployment (`socket.rc == 0`)
-        - mon_group_name in group_names
-        - containerized_deployment
+        - mon_group_name in group_names 
```

Comment:
trailling space

Reply:
Sorry, I removed the space and force-pushed this change.